e2e-tests/development-runtime/cypress/integration/gatsby-preview/refreshing.js

```diff
@@ -0,0 +1,63 @@
+const reset = () => cy.exec(`npm run reset:preview`)
+
+beforeEach(() => {
+  cy.exec(`npm run reset:preview`)
+  cy.visit(`/preview`).waitForRouteChange()
+})
+
+after(() => reset())
+
+// const update = (times = 1) =>
```

Comment:
Do these need un-commenting or removing?

Reply:
They need uncommenting. The plan is to pair with @wardpeet to understand how to copy over some fake data for testing, but we haven't been able to sync up yet since I returned from PTO.